plugins/value-container-assignment/testData/codegen/otherOperators.kt

```diff
@@ -0,0 +1,154 @@
+annotation class ValueContainer
+
+@ValueContainer
+data class StringProperty(var v: String) {
+    fun assign(v: String) {
+        this.v = v
+    }
+    fun assign(v: StringProperty) {
+        this.v = v.get()
+    }
+    fun get(): String {
+        return v
+    }
+}
+
+data class Task(val input: StringProperty)
+
+var result = "Fail"
+operator fun StringProperty.plusAssign(v: String) {
```

Comment:
Actually there is no much sense in checking all `X=` operators, because all of them are treated by same code piece. It's enough to check just one, e.g. `+=`

Reply:
Removed.